playbooks/roles/minishift/tasks/main.yml

```
@@ -1,6 +1,11 @@
 ---
 # setup minishift
 
+- name: "Running cleanup pre-tasks {{ minishift_pre_hooks }}"
```

I'm guessing this should be 
```suggestion
- name: "Running minishift pre-tasks {{ minishift_pre_hooks }}"
```